### PR TITLE
Add comments for transcript formatter

### PR DESCRIPTION
**Pull request by helias22**
### Change types

- [x] Non-breaking change

### Testing
- [ ] I have added testing to this PR

### CODE REVIEW

**Review by KayleeLChan — APPROVED** [review]
mmm spicy communication in the code it's too good. 💃 💃 💃

File: front-end/package-lock.json [hunk 1]
```diff
@@ -35,7 +35,6 @@
         "lodash": ">=4.17.21",
         "moment": ">=2.29.4",
         "postcss": "^8.4.17",
-        "minimatch": ">=3.0.5",
```
Comment (WeissSa) [inline front-end/package-lock.json:38]: We should not be removing this.

**Review by WeissSa — CHANGES_REQUESTED** [review]
Some small changes would be good. TY for commenting this!

**Review by WeissSa — APPROVED** [review]
Great changes!

**Pull request merged**